src/HttpHeaders.php

```diff
@@ -170,6 +182,7 @@ public function checkIfNotModified()
      */
     public function sendNotModifiedHeader()
     {
+        http_response_code(304);
```

Comment:
i thought we were sending 304 header in the part where we use this

Reply:
Due to the nature of PHP SAPI sometimes setting HTTP header is not enough, especially for CLI mode. I've found recommendation about this workaround and it just worked. I suggest to move to PSR-7 HTTP Messages and remove direct manipulating of the HTTP headers in the next major release.